db/migrate/20200504225017_create_events.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateEvents < ActiveRecord::Migration[6.0]
+  def change
+    create_table :events do |t|
+      t.jsonb :body
```

Comment:
could be added `null: false` here, and also a `default: {}`, so that later it does not explode with **undefined method nil**